doc/cephfs/file-layouts.rst

```diff
@@ -20,26 +20,38 @@ Layout fields
 -------------
 
 pool
-    String, giving ID or name. String can only have characters in the set [a-zA-Z0-9\_-.]. Which RADOS pool a file's data objects will be stored in.
+    This is a string and returns either an ID or a name. Strings may contain
+    only characters in the set ``[a-zA-Z0-9\_-.]``. This determines the RADOS
+    pool that stores a file's data objects.
 
 pool_id
-    String of digits. This is the system assigned pool id for the RADOS pool whenever it is created.
+    This is a string of digits. This is the pool ID that was assigned by Ceph
+    at the time of the creation of the RADOS pool.
 
 pool_name
-    String, given name. This is the user defined name for the RADOS pool whenever user creates it.
+    This is a string. This is the name of the RADOS pool as defined by the user
+    when the pool was created. 
 
 pool_namespace
-    String with only characters in the set [a-zA-Z0-9\_-.].  Within the data pool, which RADOS namespace the objects will
-    be written to.  Empty by default (i.e. default namespace).
+    This is a string containing only characters in the set ``[a-zA-Z0-9\_-.]``.
+    This determines which RADOS namespace within the data pool that the objects
+    will be written to.
+    Empty by default (i.e. default namespace).
 
 stripe_unit
-    Integer in bytes.  The size (in bytes) of a block of data used in the RAID 0 distribution of a file. All stripe units for a file have equal size. The last stripe unit is typically incomplete–i.e. it represents the data at the end of the file as well as unused “space” beyond it up to the end of the fixed stripe unit size.
+    This is an integer. The size (in bytes) of a block of data used in the
+    distribution of a file. All stripe units for a file have equal size. The
+    last stripe unit is typically incomplete–that is, it represents the data at
```

Comment:
last stripe unit is typically only partly full of data:  it holds file data data through EOF as well as padding to fill the balance of the fixed stripe unit size.

Reply:
For a future revision

Reply:
https://github.com/ceph/ceph/pull/59167